# Enforces a newline after use strict

Some style guides require that a newline follow 'use strict'.

Here are some examples:

```js
// Bad
'use strict';
var foo = 'bar';

// Good
'use strict';

var foo = 'bar';
```

## When Not To Use It

You can turn this rule off if you are not concerned about a newline after 'use strict'.
